kentikapi/errors/errors.go

```diff
@@ -0,0 +1,231 @@
+package kentik_errors
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+type Code int
+
+const (
+	AuthError          Code = 0
+	InvalidRequest     Code = 1
+	NotFound           Code = 2
+	RateLimitExhausted Code = 3
+	Temporary          Code = 4
+	Timeout            Code = 5
+)
+
+type KentikError struct {
+	Codes []Code
+	Msg   string
+}
+
+func (ke KentikError) Error() string {
+	return fmt.Sprintf("category: %v, message: %s", ke.Codes, ke.Msg)
+}
+
+func (ke KentikError) GetKentikError() *KentikError { return &ke }
```

Comment:
IMO this method does not add any value. Such methods are useful, when we have multiple error types being in the same category, e.g. Temporary(), but we are using error type, so this is not the case. Please remove that method

In our internal errutil code we can just look for KentikError type in error chain directly.

Reply:
Done